workspaces/liferay-sample-workspace/client-extensions/liferay-sample-commerce-payment-integration-sb/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+spring.config.import=\
+    classpath:/application-default.properties,\
+    optional:configtree:${LIFERAY_ROUTES_CLIENT_EXTENSION}/,\
+    optional:configtree:${LIFERAY_ROUTES_DXP}/
+
+com.liferay.lxc.dxp.domains=localhost:8080
```

Comment:
These are not needed.  This are injected now in the filesystem at the env vars listed above.